### ISSUE TITLE
예제가 2장 10장 11장 12장 13장에만 있고 나머지 장에는 없습니다.

### ISSUE THREAD

**Issue by docuhut** [docuhut/webdataviz#15]
[예제]가 2장 10장 11장 12장 13장에만 있고 나머지 장에는 없습니다.

**Comment by youhj** [comment]
네, 맞습니다. "실습"으로 만들어진 것과 "예제"로 만들어진 것에 본질적인 차이가 없습니다. 실습은 시작과 끝을 명시하는 형식적인 장치가 없는 것이 차이입니다. 추후에는 실습이 어디부터 어디까지인지 예제처럼 단위화할 필요가 있다고 생각됩니다. 

그리고 실습/예제는 사이드로 빼지 않고 본문에 포함되는 것이 좋겠습니다. 이러한 책의 성격상 이론이 있고 예제가 부가적인 것이 아니라 예제 자체가 본문의 가장 중요한 요소입니다. 예제가 본문에 있어야 합니다. 오히려 현재 본문에 있는 표로 제시된 레러펀스적인 요소들이 옆으로 빠진다면 빠질 수 있습니다.
